### ISSUE TITLE
Need a way to assign a group as a 'related group' to a set of rooms in bulk

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#5627]
not really sure how

**Comment by turt2live** [comment]
Isn't this already possible?
![image](https://user-images.githubusercontent.com/1190097/32926547-151265a0-cb05-11e7-9588-accc8e8db52e.png)

**Comment by ara4n** [comment]
my bad for the ambiguous bug. the thing missing is to say “for this set of rooms, the following flair is enabled”. the obvious thing to do would be to provide some UI on a group to bulk edit related rooms (if you have permission) for that group’s rooms.

Use case: set the 4 supporters groups as related to all rooms in the +matrix:matrix.org group. I had to do it manually and it took forever.

**Comment by lampholder** [comment]
I tried to make the title match the ask better (not sure I succeeded).

Adding a room to a group should add the group to said room's related groups list automatically if the group admin is also a room admin, so at least in what I _believe_ to be the common case the need for this should be diminished.

The room list on the group homepage could show if a room is honouring that group's flair, and potentially provide a toggle to users who have the right power levels in the room?
